dev-python/respx/respx-0.21.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Mock HTTPX with awesome request patterns and response side effects"
+HOMEPAGE="
+	https://lundberg.github.io/respx/
+	https://pypi.org/project/respx/
+	https://github.com/lundberg/respx/
+"
+# no tests in pypi sdist
+SRC_URI="
+	https://github.com/lundberg/respx/archive/refs/tags/${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/httpx[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	test? (
+		${RDEPEND}
+		dev-python/flask[${PYTHON_USEDEP}]
+		dev-python/httpcore[${PYTHON_USEDEP}]
+		dev-python/starlette[${PYTHON_USEDEP}]
+		dev-python/trio[${PYTHON_USEDEP}]
+	)
+"
+
+distutils_enable_tests pytest
+
+python_test() {
+	epytest -p no:* -o addopts=
```

Comment:
Wait, you can do it like this? i.e. don't have to do the whole `PYTEST_DISABLE_PLUGIN_AUTOLOAD` thing? In any case, please quote it, so `*` doesn't accidentally glob something.

Reply:
In fact setting `PYTEST_DISABLE_PLUGIN_AUTOLOAD` wasn't good enough for this test for whatever reason.

https://github.com/lundberg/respx/blob/master/tests/test_mock.py#L369

Reply:
Oh, perhaps `-p no:*` also enables "builtin" plugins.